internal/pkg/cli/app_status.go

```diff
@@ -24,7 +24,7 @@ const (
 type appStatusVars struct {
```

Comment:
Do we not need to rename the struct and the constants above? to `svcStatus*`

Reply:
i'll do the status renaming in my next pr :) for this pr I just fix those necessary otherwise it'll throw errors

Reply:
Oh my bad i though the PR was for app status instead of app logs lol 🤦